feature-utils/poly-look/src/css/radius.css

```diff
@@ -0,0 +1,12 @@
+:root {
+    /* Border radius */
+      --pl-radius-1x: 4px;
```

Comment:
I still don't know how I feel about this. This is going back to the conversation that we had during the workshop. My concern about having duplicated stuff still stands. Currently that's what I'm seeing. If the root problem is that `--gdu` is confusing and we are looking at `global`/`foundation` then simply renaming `gdu` to either of those solves the problem for me.

The real question is if `--pl-radius-1x` and `--pl-space-1x` will ever be out of sync, so one will be 3px and the other 5px, for eg. If the answer is no, then I don't see what we are gaining from creating all this variables.

Reply:
It is very important we all understand why we do this. and you are right, these foundations don't show that because they are just a small piece of the puzzle. 

I would recommend using the banner to test this out.  
Because using global tokens in components is not the end goal, it's a start to start thinking this way. In a systematic way. 
What is the process here? If we are blocked for just creating these variables we cannot wait for another team event. 

None of this is useful when creating a single component, but it's the same reasoning on why we use components and we don't write everything from scratch every time. For consistency and so that it can be reused, and to avoid mistakes. 

And always keep in mind that these token is bigger than just web, they will be used in all platform (android, iOs, web, figma, and any other platform we develop for) and in all kinds of projects. We could end up using these foundations also for our websites, in the polyPedia UI, anywhere where polypoly operates.

Reply:
I'm sorry, should have approved the PR before going for lunch. Like I said, I don't want to block anybody. 

My concern is that, when we are all done, we'll have bloated css that will take just as long to update, the only difference being that it won't be on the developers to do it, but on the designers.

Also, I don't want to wait for the next team event. I want to find half a day or a full day for everybody interested/affected by this asap and get the figma integration going, do the banner component, or any other component and **then** debate on how far we are going to go with this.

Reply:
What I was trying to achieve in this PR is creating some variables that we _know_ that we are going to use when those tokens are created. These variables must have the **same name** that the ones the designers will generate for us, so we could start using them ASAP and don't need to change/replace so many stuff in the future. - I should have explained this in the PR description, my bad. - Trying to save future work, basically 😄 

A mistake from my side was assuming that we were all at the same point with this (and now I see that we are not). So, I have added the `design-tokes.css` file back and moved the new ones in a separated directory to differentiate them easily. I have added the new variables to two components in poly-look as examples: `PolyButton` and `Banner`, but we will use those variables in other components when we all come to an agreement.

However, since the purpose of this PR was to start using the variables already, I am not sure what to do with this (_to merge or not to merge_). @g3o314 Do you think it will be confusing to have those variables there if we are not using them yet (so, **wait to merge until we all agree**)? Or, on the other hand, it's good to have them there as some kind of example (so, **merge**)?

Reply:
Merge :D

Also, to be clear, I'm not against design tokens at all. I want them.
Where I'm feeling unsure is when it comes to how granular we will go with them, but without a practical implementation we won't know.